RingPublishingTracking/src/main/java/com/ringpublishing/tracking/internal/util/ContentMeta+DX.kt

```diff
@@ -11,5 +11,9 @@ import com.ringpublishing.tracking.data.ContentMetadata
 internal fun ContentMetadata.buildToDX(): String
 {
 	val paid = if (paidContent) "t" else "f"
+    if (publicationId.isEmpty())
```

Comment:
krzywo?

Reply:
Nie, to tak na review tylko jakoś dziwnie widać